storage/blob.go

```diff
@@ -52,8 +87,40 @@ type Blobs interface {
 	Create(ctx context.Context, ref BlobRef, size int64) (BlobWriter, error)
 	// Open opens a reader with the specified namespace and key
 	Open(ctx context.Context, ref BlobRef) (BlobReader, error)
+	// OpenSpecific opens a reader for the already-located blob, avoiding the potential need
+	// to check multiple storage formats to find the blob.
+	OpenSpecific(ctx context.Context, ref BlobRef, formatVer FormatVersion) (BlobReader, error)
 	// Delete deletes the blob with the namespace and key
 	Delete(ctx context.Context, ref BlobRef) error
+	// Lookup looks up disk metadata on the blob file
+	Lookup(ctx context.Context, ref BlobRef) (StoredBlobAccess, error)
```

Comment:
I'm sorry for all the nitpicks on naming. Feel free to ignore if this just isn't helpful. Here `Lookup` feels a little unspecific, I wonder if `LookupBlobAccess` or something similar would be better?

Reply:
I feel like `LookupBlobAccess` implies it's finding a BlobAccess, as opposed to looking up a Blob and then providing a StoredBlobAccess around it. Maybe `LookupBlob`?

Reply:
Looking at it again, I don't like `LookupBlob` because of the confusing parallel with `Open`. I wouldn't want to change `Open` to `OpenBlob`, because `Open` is pretty standard and makes sense on a "blob store" thing. Given that, `Lookup` by itself seems best.

Reply:
Hmm, maybe `Stat` and return value of `BlobInfo`, to mimick the file access?

Reply:
Ok, I like that.